src/main/java/com/synopsys/integration/alert/channel/email/actions/EmailGlobalTestAction.java

```diff
@@ -76,7 +76,7 @@ public MessageResult testConfig(String configId, String destination, FieldAccess
                                                      .applyNotificationId(1L);
 
         ProviderMessageContent.Builder builder = new ProviderMessageContent.Builder()
-                                                     .applyProvider("Test Provider")
+                                                     .applyProvider("Test Provider", -1L)
```

Comment:
We should be able to get a real value for this at this point.

Reply:
This is the Global configuration of the email so we don't want any provider specific data to test the email channel's global configuration.